xsd/netex_framework/netex_reusableComponents/netex_trainElement_support.xsd

```diff
@@ -96,9 +95,9 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Reference to a COMPOUND TRAIN.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:complexType>
-			<xsd:complexContent>
+			<xsd:simpleContent>
```

Comment:
Why is this changed in this pull request?

Reply:
no idea how this happened..... can undo it.

Reply:
done